src/chi/javascript/components/mobile-navigation.js

```diff
@@ -0,0 +1,458 @@
+            Util.removeClass(menuItemSubItemToClose, chi.classes.ACTIVE);
+          }
+        }
+      }
+    }
+  }
+
+  _closeTabsList(menuItem) {
+    const menuItemTabs = menuItem.querySelector(`.${DRAWER_ITEM_LIST_CLASS}`);
+
+    if (Util.hasClass(menuItem, chi.classes.EXPANDED)) {
+      Util.removeClass(menuItem, chi.classes.EXPANDED);
+    }
+
+    if (menuItemTabs) {
+      const activeSubtab = menuItemTabs.querySelector(`a.${chi.classes.ACTIVE}`);
+
+      Util.removeClass(activeSubtab, chi.classes.ACTIVE);
+      menuItemTabs.style.removeProperty('height');
+    }
+  }
+
+  _activateSecondLevelMenuItem(menuItemActivator, drawerMenuItem = null, drawerId = '') {
+    const firstLevelItemToDeactive = this._mobileNavElement.querySelector(`.${FIRST_LEVEL_DRAWER_CLASS} .${DRAWER_CONTENT_CLASS} ul.${CHI_TABS_CLASS} > li.${chi.classes.ACTIVE}`);
+    const _removeSecondLevelActive = () => {
+      const activeSecondLevel = this._getActiveSecondLevelItem();
+
+      if (activeSecondLevel) {
+        this._closeTabsList(activeSecondLevel);
+        Util.removeClass(activeSecondLevel, chi.classes.ACTIVE);
+      }
+    };
+
+    if (firstLevelItemToDeactive) {
+      if (drawerId !== firstLevelItemToDeactive.getAttribute('href')) {
+        const firstLevelItemToActive = this._mobileNavElement.querySelector(`.${FIRST_LEVEL_DRAWER_CLASS} .${DRAWER_CONTENT_CLASS} ul.${CHI_TABS_CLASS} > li a[href='#${drawerId}']`);
+
+        Util.removeClass(firstLevelItemToDeactive, chi.classes.ACTIVE);
+        Util.addClass(firstLevelItemToActive.parentNode, chi.classes.ACTIVE);
+      }
+    }
+
+    if (Util.hasClass(menuItemActivator, DRAWER_ITEM_TAB_CLASS)) {
+      const currentlyActiveSecondLevelItemSubtab = this._getActiveSecondLevelItemSubtab();
+      const activeSecondLevelItem = this._getActiveSecondLevelItem();
+
+      this.drawerMenuItemRemoveUnselected();
+      if (activeSecondLevelItem && !activeSecondLevelItem.contains(menuItemActivator)) {
+        Util.removeClass(this._getActiveSecondLevelItem(), chi.classes.ACTIVE);
+      }
+      Util.addClass(menuItemActivator, chi.classes.ACTIVE);
+      Util.addClass(drawerMenuItem, chi.classes.ACTIVE);
+      if (currentlyActiveSecondLevelItemSubtab && currentlyActiveSecondLevelItemSubtab !== menuItemActivator) {
+        Util.checkRemoveClass(currentlyActiveSecondLevelItemSubtab, chi.classes.ACTIVE);
+      }
+    } else {
+      const menuItem = menuItemActivator.parentNode;
+
+      _removeSecondLevelActive();
+      Util.addClass(menuItem, chi.classes.ACTIVE);
+    }
+  }
+
+  _handleClickOnSecondLevelDrawer(e) {
+    let drawer, activator, drawerMenuItem;
+
+    if (Util.checkHasClass(e.target, chi.classes.CLOSE) ||
+      Util.checkHasClass(e.target.parentNode, chi.classes.CLOSE) ||
+      Util.checkHasClass(e.target.parentNode.parentNode, chi.classes.CLOSE)) {
+      this._hideMobileNav();
+    }
+
+    for (let cur = e.target; cur && cur !== this._drawersContainer; cur = cur.parentNode) {
```

Comment:
Does _cur_ refer to _current_? Please, use more significative names for variables.